### PR TITLE
 Jan 2020 - java - storage release details

### PR DESCRIPTION
**Pull request by hemanttanwar**

### CODE REVIEW

File: releases/2020-01/java.md [hunk 1]
```diff
@@ -137,23 +137,21 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 - Updated dependency. For more details. please see the [KeyVault Certificates](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md#400-2020-01-07)
 
 ### Storage (Blobs, Files, Queues)
-- Optimized `downloadToFile` to avoid an unnecessary `getProperties` call and to lock on an ETag once the operation has
-  started. For more details, please see the [Storage
-  Blob](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-blob/CHANGELOG.md#version-1210-2019-12-??)
+- Control data size with `ParallelTransferOptions `, Improved performance of `BlockBlobOutputStream` and llow users to provide parallel transfer options using `BlockBlobClient`. Please see the [Storage
```
Comment (gapra-msft) [inline releases/2020-01/java.md]: Typo-allow

**Comment by gapra-msft** [comment]
For the main packages(queue, blob, file share, file data lake) I would add that there was a SAS refactor adding generateSas methods on clients to make it more discoverable to the user

**Comment by adrianhall** [comment]
This still reads badly.  The change log is linked from the "latest releases" section.  Just remove the bullets that only mention the change log (which is what I was trying to convey the first comment - sorry if it was unclear)

File: releases/2020-01/java.md [hunk 2]
```diff
@@ -137,23 +137,21 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
+  Batch](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-blob-batch/CHANGELOG.md#1220-2020-01-08)
   change log.
-- Added a check in `EncryptedBlobClientBuilder` to enforce HTTPS for bearer token authentication. For more details,
-  please see the [Storage Blob
-  Cryptography](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-blob-cryptography/CHANGELOG.md#version-1210-2019-12-??)
+- For more details, please see the [Storage Blob
+  Cryptography](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-blob-cryptography/CHANGELOG.md#1220-2020-01-08)
   change log.
-- Fixed bug in ClientBuilders that prevented OAuth from functioning. For more details, please see the [Storage File Datalake](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-file-datalake/CHANGELOG.md#version-1200-beta7-2019-12-??)
+- For more details, please see the [Storage File Datalake](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-file-datalake/CHANGELOG.md#1200-beta9-2020-01-08)
```
Comment (adrianhall) [inline releases/2020-01/java.md]: Remove this bullet

File: releases/2020-01/java.md [hunk 3]
```diff
@@ -137,23 +137,21 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
   change log.
-- Renamed FileReference to StorageFileItem. For more details, please see the [Storage File Share](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-file-share/CHANGELOG.md#version-1200-preview5-2019-10-??)
+- For more details, please see the [Storage File Share](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-file-share/CHANGELOG.md#1210-2020-01-08)
```
Comment (adrianhall) [inline releases/2020-01/java.md]: Remove this bullet

File: releases/2020-01/java.md [hunk 4]
```diff
@@ -137,23 +137,21 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
   change log.
-- Added a check in ClientBuilders to enforce HTTPS for bearer token authentication. For more details, please see the [Storage Queue](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-queue/CHANGELOG.md#version-1210-2019-12-??)
+- For more details, please see the [Storage Queue](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-queue/CHANGELOG.md#1220-2020-01-08)
```
Comment (adrianhall) [inline releases/2020-01/java.md]: Remove this bullet

File: releases/2020-01/java.md [hunk 5]
```diff
@@ -137,23 +137,21 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
-- Changed exception throwing to throw `StorageBlobException` on invalid request and `BlobBatchStorageException` when
-  batch operations fail. For more details, please see the [Storage Blob
-  Batch](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-blob-batch/CHANGELOG.md#version-1210-2019-12-??)
+- For more details, please see the [Storage Blob
+  Batch](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-blob-batch/CHANGELOG.md#1220-2020-01-08)
   change log.
-- Added a check in `EncryptedBlobClientBuilder` to enforce HTTPS for bearer token authentication. For more details,
-  please see the [Storage Blob
-  Cryptography](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-blob-cryptography/CHANGELOG.md#version-1210-2019-12-??)
+- For more details, please see the [Storage Blob
```
Comment (adrianhall) [inline releases/2020-01/java.md]: Remove this bullet.

File: releases/2020-01/java.md [hunk 6]
```diff
@@ -137,23 +137,21 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
-  started. For more details, please see the [Storage
-  Blob](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-blob/CHANGELOG.md#version-1210-2019-12-??)
+- Adding generateSas methods on all the storage clients to make it more discoverable to the user.
+- Control data size with `ParallelTransferOptions`, Improved performance of `BlockBlobOutputStream` and allow users to provide parallel transfer options using `BlockBlobClient`.  For more details, Please see the [Storage
+  Blob](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-blob/CHANGELOG.md#1220-2020-01-08)
   change log.
-- Changed exception throwing to throw `StorageBlobException` on invalid request and `BlobBatchStorageException` when
-  batch operations fail. For more details, please see the [Storage Blob
-  Batch](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-blob-batch/CHANGELOG.md#version-1210-2019-12-??)
+- For more details, please see the [Storage Blob
```
Comment (adrianhall) [inline releases/2020-01/java.md]: Remove this bullet.

File: releases/2020-01/java.md [hunk 7]
```diff
@@ -137,23 +137,21 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 - Updated dependency. For more details. please see the [KeyVault Certificates](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md#400-2020-01-07)
 
 ### Storage (Blobs, Files, Queues)
-- Optimized `downloadToFile` to avoid an unnecessary `getProperties` call and to lock on an ETag once the operation has
-  started. For more details, please see the [Storage
-  Blob](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-blob/CHANGELOG.md#version-1210-2019-12-??)
+- Adding generateSas methods on all the storage clients to make it more discoverable to the user.
+- Control data size with `ParallelTransferOptions`, Improved performance of `BlockBlobOutputStream` and allow users to provide parallel transfer options using `BlockBlobClient`.  For more details, Please see the [Storage
+  Blob](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.2.0/sdk/storage/azure-storage-blob/CHANGELOG.md#1220-2020-01-08)
```
Comment (adrianhall) [inline releases/2020-01/java.md]: Remove the "For more details, see the changelog" comment.

**Pull request merged**